src/TSTransformer/util/hasSymbolProperty.ts

```diff
@@ -0,0 +1,7 @@
+import ts from "typescript";
+
+export function hasSymbolProperty(type: ts.Type, symbolName: string) {
```

Comment:
Would it be useful to make this `string` a specific list of allowed Symbol kinds? (To prevent typos etc)
Alternatively and/or additionally, maybe this can be asserted at runtime using the list that `ts.isKnownSymbol` uses internally? Unless I'm misunderstanding what that does.